WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -11,7 +11,10 @@ class AztecPostViewController: UIViewController {
     static let margin = CGFloat(20)
 
     private(set) lazy var richTextView: Aztec.TextView = {
-        let tv = Aztec.TextView(defaultFont: WPFontManager.merriweatherRegularFontOfSize(16))
+        let defaultFont = WPFontManager.merriweatherRegularFontOfSize(16)
+        // TODO: Add a proper defaultMissingImage
```

Comment:
It was more just for me but I believe @diegoreymendez is going to be cleaning up the Aztec issues here.